test/conformance/runtime/container_test.go

```diff
@@ -43,8 +43,7 @@ func TestMustNotContainerConstraints(t *testing.T) {
 		name: "TestArbitraryPortName",
 		options: func(s *v1alpha1.Service) {
```

Comment:
And there should be a test here that sets the port.

Reply:
I think all the MUST and SHOULD clauses from the [runtime contract](
https://github.com/knative/serving/blob/master/docs/runtime-contract.md#protocols-and-ports) are already covered here? cc @dgerd

Reply:
Actually, I think you're right:

>  Only one inbound containerPort SHALL be specified in the core.v1.Container specification.

We test for more than one, but we aren't testing for one

Reply:
Nevermind, it gets tested in https://github.com/knative/serving/blob/master/test/conformance/api/v1alpha1/service_test.go#L37